inductiva/tasks/README.md

```diff
@@ -0,0 +1,124 @@
+# Task management
+
+
+### Using the `Task` class
+
+As shown in the [scenarios](../../README.md#scenarios) examples, both the sync and [async](../../README.md#async-api) runs of the scenarios shown above return a `Task` object.
+This object provides methods for managing a specific task submitted to the **Inductiva API**.
+For instance, you can:
+ * Get its status;
+ * Get the machine type where it ran/is running;
+ * Kill it if you've changed your mind;
+ * Download output files (all of them or only the important ones);
+ * Get its execution time.
+
+Check out some example usage of `Task`:
+
+```python
+# `scenario` constructed as in the examples above.
+# ... stands for arguments related to each specific scenario.
+task = scenario.simulate(..., run_async=True)
+
+# Get status of the task.
+status = task.get_status()
+print(status) # would be, e.g., "submitted", "started", "success", "failed", "killed"
```

Comment:
Should we have some section in the docs where we explain what is each of these states, and what the user should expect in each one? Ideally, we would have an image explaining the flow, but we can leave that for later.

In this section, you could explain, for example, why tasks can fail (do we already have a few documented cases?)